File: CS/dxSample/Form1.cs
using dxSample.Data;

namespace dxSample {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {

        public Form1() {
            InitializeComponent();
            regularTemplateListBox.ItemAutoHeight = true;
            htmlTemplateListBox.ItemAutoHeight = true;

            categoriesBindingSource.DataSource = CategoryData.GetCategories();
        }   
    }
}

File: VB/dxSample/Form1.vb
Imports dxSample.Data

Namespace dxSample

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            regularTemplateListBox.ItemAutoHeight = True
            htmlTemplateListBox.ItemAutoHeight = True
            categoriesBindingSource.DataSource = GetCategories()
        End Sub
    End Class
End Namespace
